composer.json

```diff
@@ -1,54 +1,52 @@
 {
-	"name":"friendsofcake/crud-view",
-	"version": "0.0.1",
-	"description":"View layer for CRUD",
-	"type":"cakephp-plugin",
-	"keywords":[
-		"cakephp",
-		"crud",
-		"create",
-		"retrieve",
-		"update",
-		"delete",
-		"bake",
-		"cake",
-		"scaffold",
-		"scaffolding"
-	],
-	"homepage":"https://github.com/FriendsOfCake/crud-view",
-	"license":"MIT",
-	"authors":[
-		{
-			"name":"Christian Winther",
-			"role":"Author",
-			"homepage":"http://cakephp.nu/"
-		}
-	],
-	"require":{
-		"friendsofcake/crud":"*"
-	},
-	"require-dev": {
-		"cakephp/cakephp": "3.0.*-dev",
-		"cakephp/app": "dev-master",
-		"phpunit/phpunit": "3.7.33",
+    "name":"friendsofcake/crud-view",
+    "version": "0.0.1",
+    "description":"View layer for CRUD",
+    "type":"cakephp-plugin",
+    "keywords":[
+        "cakephp",
+        "crud",
+        "create",
+        "retrieve",
+        "update",
+        "delete",
+        "bake",
+        "cake",
+        "scaffold",
+        "scaffolding"
+    ],
+    "homepage":"https://github.com/FriendsOfCake/crud-view",
+    "license":"MIT",
+    "authors":[
+        {
+            "name":"Christian Winther",
+            "role":"Author",
+            "homepage":"http://cakephp.nu/"
+        }
+    ],
+    "require":{
+        "friendsofcake/crud":"*",
+        "friendsofcake/bootstrap-ui": "~0.3"
+    },
+    "require-dev": {
+        "cakephp/cakephp": "3.0.*-dev",
+        "cakephp/app": "dev-master",
+        "phpunit/phpunit": "3.7.33",
```

Comment:
Why? It's a development dependency.

Note: all i did was reformat this file really (and add bootstrap-ui).